scripts/check-test-service.ts

```diff
@@ -0,0 +1,9 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
Minor: Perhaps you can adjust the name of the script file to something like: `check-test-service-for-changes.ts` then it is clear from the name what the script does.